tests/commands/dev/test_get_environment.py

```diff
@@ -1,12 +1,29 @@
 import sys
 from pathlib import Path
+from unittest.mock import PropertyMock
 
 import pytest
 
 PYTHONPATH = "PYTHONPATH"
 PYTHONMALLOC = "PYTHONMALLOC"
 
 
+@pytest.mark.parametrize("platform", ["windows", "darwin", "linux", "wonky"])
+def test_pythonmalloc_only_set_for_windows(
+    dev_command,
+    first_app,
+    platform,
+    monkeypatch,
+):
+    """PYTHONMALLOC env var is only set on Windows."""
+    monkeypatch.setattr(
+        type(dev_command), "platform", PropertyMock(return_value=platform)
+    )
+    env = dev_command.get_environment(first_app, test_mode=False)
+    expected = "default" if platform == "windows" else "missing"
+    assert env.get(PYTHONMALLOC, "missing") == expected
```

Comment:
I'm not 100% sold on the approach here. There's effectively 2 options:

1. Mark the branch as no-cover, and only test it on Windows
2. monkeypatch platform and test the branch on every platform.

My inclination is that it's better to do the former, as our mock of a platform is more likely to be in error than coverage's identification of the platform.

Reply:
Unless I'm not following entirely, I did implement option 2. I switched to option 1, though. I had to use `# pragma: no cover` since no platform will take both branches.